src/app/personside/infotabs/oversikt/OppfolgingOversikt.tsx

```diff
@@ -43,11 +43,13 @@ function YtelserForBruker({ detaljertOppfolging }: { detaljertOppfolging: Detalj
     if (detaljertOppfolging.ytelser.length === 0) {
         return null;
     }
-    const ytelser = detaljertOppfolging.ytelser.map(ytelse => ytelse.type);
+    const ytelser = detaljertOppfolging.ytelser
+        .filter(ytelse => ytelse.status !== 'Avsluttet')
+        .map(ytelse => ytelse.type + ' : ' + ytelse.status);
     const filtrerteYtelser = ytelser.filter((item, index) => ytelser.indexOf(item) === index).join(', ');
     return (
         <>
-            <Element>Ytelser:</Element>
+            <Element>{filtrerteYtelser.length > 0 ? 'Ytelser:' : ''}</Element>
```

Comment:
Om vi skal fjerne "Ytelser: " overskriften ville jeg vurdert å bare returne `null` fra komponenten. 
På den måten slipper vi at det kommer to tomme `<p></p>` ut og forvirrer skjermlesere.

Evt kan vi bare la overskriften stå.